RELEASES.md

```diff
@@ -4,68 +4,72 @@
 
 Due to the period of time between this release and v0.0.26, many relevant
 changes and fixes may not appear in the following list.  For a complete list of
-changes, please check: https://github.com/google/kythe/commits/
+changes, please check the commit logs:
+https://github.com/google/kythe/search?q=committer-date%3A2016-11-01..2018-05-31&type=Commits
```

Comment:
We will need to update the dates here, once we pick a release date. :)

Reply:
`https://github.com/google/kythe/compare/v0.0.26...v0.0.27`

Reply:
Oh, nice, thanks. Done.